### PR TITLE
Julia

### PR DESCRIPTION
**Pull request by fruitl00p**
Got Julia to run.

Had to update the raffler in order to have the random function actually work... (might be due to the version differences?)

### CODE REVIEW

**Comment by fruitl00p** [comment]
sigh... i suck at branching btw... ;)

File: drvanr-fortran/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Alpine's gfortran wouldnt compile ?
+FROM debian:jessie
+MAINTAINER robin@kingsquare.nl
+
+# Create working dir
+RUN mkdir -p /var/app
+COPY ./raffler.F /var/app
```
Comment (lucasvanlierop) [inline drvanr-fortran/Dockerfile]: You have specify the copied file, like: `COPY ./raffler.F /var/app/raffler.F`

Comment (fruitl00p) [inline drvanr-fortran/Dockerfile]: It does work though? I only changed the randomness Stuff...

**Comment by fruitl00p** [comment]
@lucasvanlierop the weird thing is, my local julia works with this PR. I might clean up the PR and just include Julia based from the latest master... 

will update ;)

**Pull request closed without merging**